pkg/api/helpers.go

```diff
@@ -432,6 +432,14 @@ const (
 	// PreferAvoidPodsAnnotationKey represents the key of preferAvoidPods data (json serialized)
 	// in the Annotations of a Node.
 	PreferAvoidPodsAnnotationKey string = "scheduler.alpha.kubernetes.io/preferAvoidPods"
+
+	// AppArmorPodAnnotationKey represents the key of an AppArmor profile applied to all containers of a
+	// pod.
+	AppArmorPodAnnotationKey string = "apparmor.security.alpha.kubernetes.io/pod"
+
+	// AppArmorContainerAnnotationKeyPrefix represents the key of an AppArmor profile applied to one
+	// container of a pod.
+	AppArmorContainerAnnotationKeyPrefix string = "apparmor.security.alpha.kubernetes.io/container/"
```

Comment:
Oh, good to know. I'll change it to match the seccomp key:
```
container.apparmor.security.alpha.kubernetes.io/<name>
```
I was wondering why the seccomp key was changed from the original `/contanier/<name>`...